src/components/Microsoft.DotNet.UpgradeAssistant/TargetFramework/TargetFrameworkSelector.cs

```diff
@@ -47,7 +41,14 @@ public async ValueTask<TargetFrameworkMoniker> SelectTargetFrameworkAsync(IProje
                 throw new ArgumentNullException(nameof(project));
             }
 
-            var appBase = _upgradeTarget == UpgradeTarget.Current ? _currentTFMBase : _ltsTFMBase;
+            var appBase = _upgradeTarget switch
+            {
+                UpgradeTarget.Current => _selectorOptions.Current,
+                UpgradeTarget.Preview => _selectorOptions.Preview,
+                UpgradeTarget.LTS => _selectorOptions.LTS,
+                _ => _selectorOptions.LTS,
```

Comment:
Does this mean this defaults to LTS?

Reply:
No - this is if there's a different enum value (probably should never even happen, but just in case)

Reply:
I can't seem to find where you are defaulting to Current, could you point me to that?